packages/components/src/components/tick-picker/tick-picker.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import Button              from '../button';
+import Icon                from '../icon';
+
+const TickPicker = ({ min_value, max_value, onSubmit, label }) => {
+    const normalizedTick = (tick) => ((parseInt(tick) < 10 ? '0' : '') + tick);
+    const [tick_value, setTickValue] = useState(parseInt(min_value));
+
+    const handleDecrease = () => {
+        if (tick_value - 1 >= min_value) {
+            setTickValue(tick_value - 1);
+        }
+        
+    };
+    const handleIncrease = () => {
+        if (tick_value + 1 <= max_value) {
+            setTickValue(tick_value + 1);
+        }
+        
+    };
+    const handleClick = () => {
+        onSubmit({
+            target: {
+                value: tick_value,
+                name : 'submit',
+            },
+        });
+    };
+    return (
+        <div className='tick-picker'>
+            <div className='tick-picker--calculation'>
```

Comment:
The naming doesn't feel right, `tick-picker--calculation` in BEM refers to a modifier that might not be there. e.g. there is a `calculation` mode and that require different styling. You can use `tick-picker__calculation` to refer to `calculation` element inside `tick-picker`.

Reply:
Done